.ci/Jenkinsfile

```diff
@@ -400,11 +400,12 @@ pipeline {
     }
     stage('e2e tests') {
       when {
-        // Always when running builds on branches/tags
+        // Disable until we fix the e2e test.
+        // This should be always running when builds on branches/tags
         // Enable if e2e related changes.
         beforeAgent true
         anyOf {
-          not { changeRequest() }
+          //not { changeRequest() }
```

Comment:
I would consider just deleting this rather than leaving it commented out.

Reply:
You are right, I am thinking that this change is temporary until the e2e is fixed, so once that is done we would uncomment and re-enable them.

Reply:
But I guess we can look at the git history and git revert this PR